src/browser/squirrel-update.coffee

```diff
@@ -231,19 +129,24 @@ exports.handleStartupEvent = (app, squirrelCommand) ->
   switch squirrelCommand
     when '--squirrel-install'
       createShortcuts ->
-        installContextMenu ->
+        WinRegistry.installContextMenu ->
           addCommandsToPath ->
             app.quit()
       true
     when '--squirrel-updated'
       updateShortcuts ->
-        installContextMenu ->
+        contextMenuWasEnabled = atom.config.get('core.showAtomInShellContextMenu')
```

Comment:
I don't think `atom.config` is available in the browser process where this code runs. Did this work for you? I would have guessed this line would throw have thrown an error.